src/test/unit/agrad/fwd/matrix/stan_print_test.cpp

```diff
@@ -24,28 +24,28 @@ TEST(AgradFwdMatrixStanPrint,fvar_double) {
   b.push_back(a);
   b.push_back(a);
   stan_print(&output, b);
-  EXPECT_EQ("[1:2,1:2,1:2]",output.str());
+  EXPECT_EQ("[1,1,1]",output.str());
```

Comment:
Are we enforcing google-style guide rules for tests? If so, spaces before output.str() throughout the file. If not, ignore.